src/api/SensumApi.js

```diff
@@ -2,8 +2,8 @@
 
 /* Config */
 const config = {};
-config.serverRoute  = 'https://sensum-server.herokuapp.com/api';
-// config.serverRoute = 'http://192.168.0.161:3000/api';
+// config.serverRoute  = 'https://sensum-server.herokuapp.com/api';
+config.serverRoute = 'http://192.168.0.202:3000/api';
```

Comment:
I think you left this here by accident

Reply:
More or less, not by accident indeed.. just testing. Hope to remember revert it before merging haha! 🤞

Reply:
I'll leave these comments up but I honestly didn't notice this was WIP.  You can close the threads if you want.